### ISSUE TITLE
[Exception][PyCharm 2023.2.1] *** exception class was changed or removed

### ISSUE THREAD

**Issue by jacobuserasmus** [Codium-ai/codiumai-jetbrains-release#121]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
  A suspiciously high nesting of suppressPrioritizing, forgot to call restorePrioritizing?
  
  java.lang.Throwable
at com.intellij.idea.IdeaLogger.error(IdeaLogger.java:131)
at com.intellij.openapi.diagnostic.Logger.error(Logger.java:384)
at com.intellij.openapi.progress.impl.CoreProgressManager.suppressPrioritizing(CoreProgressManager.java:824)
at com.intellij.openapi.progress.impl.CoreProgressManager.avoidBlockingPrioritizingThread(CoreProgressManager.java:879)
at com.intellij.openapi.progress.impl.CoreProgressManager.sleepIfNeededToGivePriorityToAnotherThread(CoreProgressManager.java:840)
at com.intellij.openapi.progress.impl.ProgressManagerImpl.runCheckCanceledHooks(ProgressManagerImpl.java:192)
at com.intellij.openapi.progress.impl.CoreProgressManager.doCheckCanceled(CoreProgressManager.java:145)
at com.intellij.openapi.progress.ProgressManager.checkCanceled(ProgressManager.java:231)
at com.intellij.openapi.progress.ProgressIndicatorProvider.checkCanceled(ProgressIndicatorProvider.java:23)
at com.intellij.psi.impl.source.tree.CompositeElement.getPsi(CompositeElement.java:676)
at com.intellij.psi.impl.source.SourceTreeToPsiMap.treeElementToPsi(SourceTreeToPsiMap.java:15)
at com.intellij.psi.impl.source.tree.SharedImplUtil.getNextSibling(SharedImplUtil.java:45)
at com.intellij.psi.impl.source.tree.LeafPsiElement.getNextSibling(LeafPsiElement.java:61)
at com.intellij.psi.SyntaxTraverser$PsiApi.next(SyntaxTraverser.java:308)
at com.intellij.psi.SyntaxTraverser$PsiApi.next(SyntaxTraverser.java:264)
at com.intellij.psi.SyntaxTraverser$ApiEx$1.fun(SyntaxTraverser.java:254)
at com.intellij.util.containers.JBIterable$2$1.nextImpl(JBIterable.java:129)
at com.intellij.util.containers.JBIterator.peekNext(JBIterator.java:132)
at com.intellij.util.containers.JBIterator.hasNext(JBIterator.java:92)
at com.intellij.util.containers.TreeTraversal$BiOrderIt.nextImpl(TreeTraversal.java:498)
at com.intellij.util.containers.JBIterator.peekNext(JBIterator.java:132)
at com.intellij.util.containers.JBIterator.hasNext(JBIterator.java:92)
at com.intellij.util.containers.JBIterable.isEmpty(JBIterable.java:762)
at com.intellij.psi.util.PsiTreeUtil.hasErrorElements(PsiTreeUtil.java:1222)
at ai.codium.ui.structureview.CodiumStructureViewRootElement.getChildren(CodiumStructureViewElements.kt:85)
at ai.codium.ui.structureview.CodiumStructureViewRootElement.getChildren(CodiumStructureViewElements.kt:69)
at com.intellij.ide.util.treeView.smartTree.TreeElementWrapper.initChildren(TreeElementWrapper.java:35)
at com.intellij.ide.util.treeView.smartTree.CachingChildrenTreeNode.rebuildSubtree(CachingChildrenTreeNode.java:207)
at com.intellij.ide.util.treeView.smartTree.CachingChildrenTreeNode.ensureChildrenAreInitialized(CachingChildrenTreeNode.java:41)
at com.intellij.ide.util.treeView.smartTree.CachingChildrenTreeNode.getChildre

<TRIMMED STACKTRACE>

ctionAndCancelBeforeWrite(ProgressIndicatorUtilService.java:63)
at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:133)
at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:114)
at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:71)
at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:71)
at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:111)
at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:76)
at com.intellij.util.concurrency.Invoker.invokeSafely(Invoker.java:188)
at com.intellij.util.concurrency.Invoker.lambda$offerSafely$0(Invoker.java:165)
at com.intellij.util.concurrency.Invoker$Background.lambda$offer$0(Invoker.java:433)
at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:249)
at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:31)
at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:227)
at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:218)
at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:215)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
at java.base/java.lang.Thread.run(Thread.java:833)

```

# Other Information
```
=== About ===
Build version: PyCharm 2023.2.1 Build: #PY-232.9559.58 August 22, 2023
Theme: Darcula
JRE: 17.0.8+7-b1000.8, JetBrains s.r.o.
JVM: 17.0.8+7-b1000.8, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Linux 6.2.0-32-generic (amd64)

=== Plugins ===
Custom plugins: [XPathView + XSLT (232.8660.129), Logstash Config Files Support (0.8.3), Ideolog (222.1.0.0), CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.6.07), Odoo (1.2.1), Odoo (2023.9.0.232), ChatGPT Tool (1.1.1), HOCON (2023.1.0), Codeium: AI Autocomplete and Chat for Python, JS, TS, Java, Go… (1.2.87), Requirements (2022.4.1), GitHub Copilot (1.2.22.3129), CSV Editor (3.2.1-232), Rainbow CSV (2.0.2)]
Disabled plugins:[Datalore (0.1.14-212)]
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/9Vmj5qDh) by [Unito](https://www.unito.io)

**Comment by aldanchenko** [comment]
Hi @jacobuserasmus 

Thanks for reporting! Maybe you could help us to reproduce this issue? Could you please share steps to reproduce or any other info?

Best regards,
Alex

**Comment by ArtsiomCh** [comment]
Fixed in 0.6.12
